src/DatePicker/style.css

```diff
@@ -0,0 +1,216 @@
+.ant-picker {
+  @apply h-10 px-4 border-transparent shadow-none rounded-lg;
+  @apply hover:border-neutral-300 focus:border-neutral-300;
+
+  &-panel-container {
+    @apply rounded-lg;
+  }
+
+  &-input > input {
+    @apply text-neutral-700 font-inter;
+  }
+
+  &-header{
+    @apply border-none;
+  }
+
+  &-range {
+    .ant-picker-active-bar {
+      @apply bg-slate-300;
+    }
+  }
+
+  &-range-separator {
+    @apply px-4 text-slate-300;
+  }
+
+  &-suffix {
+    @apply text-slate-700;
+  }
+
+  &-clear > span > svg{
+    @apply h-4 w-4 text-transparent;
+    background-image: url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='16' height='16' fill='dimgray' viewBox='0 0 256 256'><rect width='256' height='256' fill='none'></rect><line x1='200' y1='56' x2='56' y2='200' stroke='dimgray' stroke-linecap='round' stroke-linejoin='round' stroke-width='16'></line><line x1='200' y1='200' x2='56' y2='56' stroke='dimgray' stroke-linecap='round' stroke-linejoin='round' stroke-width='16'></line></svg>");
+  }
+
+  &-cell-in-view.ant-picker-cell-today {
+    .ant-picker-cell-inner::before {
+      @apply rounded-full border-neutral-300;
+    }
+  }
+
+  &-cell .ant-picker-cell-inner {
+    @apply rounded-full inline-block w-auto;
+    @apply px-0;
+  }
+
+  &-year-panel .ant-picker-cell-inner {
+    @apply px-2.5 rounded-lg;
+  }
+
+  &-cell-in-view.ant-picker-cell-selected {
+    .ant-picker-cell-inner {
+      @apply bg-slate-700;
+    }
+  }
+
+  &-cell-in-view.ant-picker-cell-range-start:not(.ant-picker-cell-range-start-single)::before,
+  &-cell-in-view.ant-picker-cell-range-end:not(.ant-picker-cell-range-end-single)::before {
+    @apply bg-slate-700;
+  }
+
+  &-cell-in-view.ant-picker-cell-in-range::before {
+    @apply bg-slate-700;
+  }
+
+  &-cell-in-view.ant-picker-cell-in-range {
+    @apply text-white;
+  }
+
+  &-header-view button {
+    @apply text-[#1D3557] text-base font-inter font-bold uppercase leading-6;
+    @apply hover:opacity-80 hover:text-[#1D3557];
```

Comment:
You can use preset colors like `text-slate-700`. If the color is not included, it needs to be added in the preset repository